pd-control.md

```diff
@@ -765,7 +787,32 @@ Usage:
 - `src-tolerance-ratio` and `dst-tolerance-ratio` are configuration items for the expectation scheduler. The smaller the `tolerance-ratio`, the easier it is for scheduling. When redundant scheduling occurs, you can appropriately increase this value.
 
     ```bash
-    >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.05
+       >> scheduler config balance-hot-region-scheduler set src-tolerance-ratio 1.1
+    ```
+
+- `read-priorities`,`write-leader-priorities`, and `write-peer-priorities` control which dimensions have the priority for hot Region scheduling and support two dimensions.
+
+    - `read-priorities` and `write-leader-priorities` control the dimensions to be prioritized for read and write-leader hot Region scheduling. The available dimensions are `query`, `byte` and `key`.
+    - `write-peer-priorities` controls the dimension to be prioritized for write-peer hot Region scheduling. The available dimensions are `byte` and `key`.
+    
+    > **Note:**
+    >
+    > If not all the cluster components are upgraded to v5.2 and later, the configuration of `query` dimension does not take effect. If some components are upgraded, the `byte` and `key` dimensions still by default have the priority for hot Region scheduling. After all the components of the cluster are upgraded to v5.2 and later, such a compatible configuration still takes effect. You can view the real-time configuration using `pd-ctl` command. Usually, you do not need to modify these configurations.
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set read-priorities query,byte
+    ```
+
+- `strict-picking-store` controls the search space of hot Region scheduling. Usually it is enabled. When it is enabled, hot Region scheduling ensures the balance between the two configured dimensions. When it is disabled, hot Region scheduling only ensures the balance of dimensions with the first priority, which might reduce the balance of other dimensions. Usually, you do not need to modify this configuration.
```

Comment:
```suggestion
- `strict-picking-store` controls the search space of hot Region scheduling. When it is enabled, the scheduler performs hot Region scheduling while ensuring stability. Usually, it is enabled. When it is disabled, hot Region scheduling only ensures the balance of dimensions with the first priority, which might reduce the Region balance of other dimensions. Usually, you do not need to modify this configuration.
```

Reply:
In https://github.com/pingcap/docs-cn/pull/6888, the Chinese in this section has been improved because the author thinks `ensuring stability` makes readers confusing. PTAL~

Reply:
Thanks! I have modified my comment. PTAL again.